packages/sandbox/src/compilerOptions.ts

```diff
@@ -50,11 +50,18 @@ export function getDefaultSandboxCompilerOptions(config: SandboxConfig, monaco:
     emitDecoratorMetadata: true,
     moduleResolution: monaco.languages.typescript.ModuleResolutionKind.NodeJs,
 
-    target: monaco.languages.typescript.ScriptTarget.ES2017,
+    target: monaco.languages.typescript.ScriptTarget.ESNext,
```

Comment:
I can split this into another PR, I just know we get surprised by this all the time.

Reply:
It's probably time.

Reply:
Er. Wait. I meant lib=esnext... This is the target. Maybe this is not a good idea?

Reply:
Target implies lib, right?

Reply:
It does - if you want, you can switch to `lib: esnext`, I'm not sure if that excludes the DOM.

Reply:
Alrighty.

I also realized that this probably will make decorators not work in the browser, because the browsers don't have native support yet. So, I think I'll just revert this for now?

Touching `lib` is difficult here; I tried to do a similar thing for our tests and I think CompilerOptions is a little funky, and I'd need to specify different names than in tsconfig.